### PR TITLE
Fix duplicate metadata

### PR DESCRIPTION
**Pull request by stefanroelofs**
## Proposed change
Some metadata (like descriptions) was copy pasted without changing it.
Or it was missing completely.

Also added some headings and fixed minor typo's.

Thanks!

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/free_mobile.markdown [hunk 1]
```diff
@@ -11,21 +11,25 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `free_mobile` platform is using the French mobile operator [Free Mobile](http://mobile.free.fr/) to send SMS to your own cell phone. 
+The `free_mobile` platform uses the French mobile operator [Free Mobile](http://mobile.free.fr/) to send SMS to your own cell phone.
+
+## Setup
```
Comment (frenck) [inline source/_integrations/free_mobile.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/llamalab_automate.markdown [hunk 2]
```diff
@@ -13,9 +13,13 @@ ha_integration_type: integration
 
 The `llamalab_automate` platform uses Googles Cloud Messaging Services to push messages from Home Assistant to your Android device running the LlamaLab [Automate](https://llamalab.com/automate/) app. This can serve as an alternative to Tasker + AutoRemote.
 
+## Setup
```
Comment (frenck) [inline source/_integrations/llamalab_automate.markdown]:
```suggestion
## Prerequisites
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/free_mobile.markdown (hunk 1 above)
Comment (stefanroelofs) [reply source/_integrations/free_mobile.markdown]: The **Setup** heading is used in documentation of many (>100) integrations, like: [accuweather](https://www.home-assistant.io/integrations/accuweather) or [amazon polly](https://www.home-assistant.io/integrations/amazon_polly).

On the other hand, **Prerequisites** is also used for about 60 integrations.

I'll commit the changes.
Thanks!

Comment (frenck) [reply source/_integrations/free_mobile.markdown]: Yeah, we started using `Prerequisites` some time ago as it better describes the intent.
Setup and Configuration are generally a bit confusing, as they can mean the same (as a matter of fact, we have integrations that describe our configuration procedure with a "Setup" heading.

**Review by frenck — APPROVED** [review]
Thanks, @stefanroelofs 👍

File: source/_integrations/free_mobile.markdown (hunk 1 above)
Comment (stefanroelofs) [reply source/_integrations/free_mobile.markdown]: Very clear, and I agree. 
Thanks!

**Pull request merged**